prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -385,5 +386,55 @@ public async Task<ActionResult> sendRemoteUsersEmail(int siteId)
             return NoContent();
         }
 
+        // POST: api/Sites/5/remote-users-email-admin
+        /// <summary>
+        /// Send HIBC an email when remote users are updated for a submitted site
+        /// </summary>
+        /// <param name="siteId"></param>
+        [HttpPost("{siteId}/remote-users-email-admin", Name = nameof(sendRemoteUsersEmailAdmin))]
```

Comment:
Can only admin send emails using these two endpoints? If so, we should add the admin auth header to these.

Reply:
No it just sends an email to the admins

Reply:
And this new one sends an email to the remote user

Reply:
As is, these two endpoints can be triggered by any user to send emails to other people. Can we restrict these two endpoints in any way, similar to what we are doing on other endpoints?